pkg/cover/report.go

```diff
@@ -893,6 +913,32 @@ func archCallInsn(target *targets.Target) ([][]byte, [][]byte) {
 	}
 }
 
+func (rg *ReportGenerator) PCs() map[uint64][]symbolizer.Frame {
+	var allPCs []uint64
+	for _, s := range rg.symbols {
+		allPCs = append(allPCs, s.pcs...)
+	}
+	var progs []Prog
+	progs = append(progs, Prog{Data: "allPCs", PCs: allPCs})
+	// Pass a fake prog with all symbol pcs to preoareFileMap, rg.pcs will be initialized.
+	_, err := rg.prepareFileMap(progs)
```

Comment:
What does this do beyond symbolizing PCs? If we need to just symbolize, then it's easier to call symbolize function directly rather than prepareFileMap and then convert back to symbolizer frames.

Reply:
Considering web UI "cover" will use rg.pcs also, and call symbolize would store nothing to rg.pcs.
It seems that initializing rg.pcs will spend some time in symbolize.
If use symbolize here, some works are duplicate.
Do you think the cost is acceptable here?

Reply:
Yes, symbolization is slow. Especially whole vmlinux symbolization, it takes 10 mins for me. The whole point of the recent pkg/cover/report.go overhaul was to do incremental symbolization. And here you take all PCs and symbolize, so we get back to whole vmlinux symbolization...

Fortunately I think we don't need to symbolize for coverage filter at all!
When we have raw PC files we don't even need to initialize report generator, we already have all PCs.
For function names we could look at symbol names as these are effectively function names.
For file names we could look at symbol.unit.filename I think and select symbols this way.

Please do it this way. I will be way faster and will make people want to use it.

Reply:
I used the way you mentioned.
Since `symbols` is an unexpected export field of `ReportGenerator` ( `unit` of `symbol` is the same).
I have to add a method of `ReportGenerator`:
```  
func (rg *ReportGenerator) GetSymbolsInfo() (symName []string, cuName []string, pcs [][]uint64)
```